src/compiler/Error.elm

```diff
@@ -53,6 +53,7 @@ type ParseContext
     | InLet
     | InLetBinding
     | InLambda
+    | InUnit
```

Comment:
this looks useless, wdyt?

Reply:
I believe it can help us when debugging other parsers - I'd keep it. (But, I have to get up to speed with "how best to debug parsers" myself. That's the part that gives me the most trouble when writing them :) )